redash/query_runner/presto.py

```diff
@@ -98,14 +118,22 @@ def get_schema(self, get_stats=False):
         return schema.values()
 
     def run_query(self, query, user):
+        should_impersonate_user = self.configuration.get('user_impersonation', False)
+        if not should_impersonate_user or user is None:
```

Comment:
better add (redundant) parenthesis
`if (not should_impersonate_user) or (user is None)`